src/configuration/routes/cache.md

```diff
@@ -178,3 +176,5 @@ default value.
 If the response does not have a `Cache-Control` header, the
 `default_ttl` key is used to define the cache duration, in seconds. The
 default value is `0`.
+
+If the application code returns a Cache-Control header, or if your `.platform.app.yaml` file is configured to set a cache lifetime for static assets, then this value is ignored in favor of the application container's headers.
```

Comment:
Well, there's the subtle question of whether "the application" includes the user's code, or the user's code plus the configured nginx instance. It technically means both depending on the sentence. :smile: